docs/standard/containerized-lifecycle-architecture/index.md

```diff
@@ -7,23 +7,21 @@ ms.date: 11/23/2018
 ---
 # Introduction to containers and Docker
 
-Containerization is an approach to software development in which an application or service, its dependencies, and its configuration (abstracted as deployment manifest files) are packaged together as a container image. You then can test the containerized application as a unit and deploy it as a container image instance to the host operating system.
+*Containerization is an approach to software development in which an application or service, its dependencies, and its configuration (abstracted as deployment manifest files) are packaged together as a container image. You then can test the containerized application as a unit and deploy it as a container image instance to the host operating system (OS).*
 
-Just as the shipping industry uses standardized containers to move goods by ship, train, or truck, regardless of the cargo within them, software containers act as a standard unit of software that can contain different code and dependencies. Placing software into containers makes it possible for developers and IT professionals to deploy those containers across environments with little or no modification.
+Just as shipping containers allow goods to be transported by ship, train, or truck regardless of the cargo inside, software containers act as a standard unit of software deployment that can contain different code and dependencies. Containerizing software this way enables developers and IT professionals to deploy them across environments with little or no modification.
 
-Containers also isolate applications from one another on a shared operating system (OS). Containerized applications run on top of a container host, which in turn runs on the OS (Linux or Windows). Thus, containers have a significantly smaller footprint than virtual machine (VM) images.
+Containers also isolate applications from each other on a shared OS. Containerized applications run on top of a container host that in turn runs on the OS (Linux or Windows). Containers therefore have a much smaller footprint than virtual machine (VM) images.
 
-Each container can run an entire web application or a service, as shown in Figure 1-1.
+Each container can run a whole web application or a service, as shown in Figure 1-1. In this example, Docker host is a container host, and App1, App2, Svc1, and Svc2 are containerized applications or services.
 
-![](./media/image1.png)
+![Two applications and two services running on the OS in a VM or a physical server](./media/image1.png)
```

Comment:
another task that we'll have to do at a later time is to add meaningful file names for images including dashes to separate words. It impacts SEO.

Reply:
Yep, I started doing it for new images, I didn't understand why at first, but now I'm using it even in my blogposts! 😊

This was one of the nice takeaways from this assignment (besides learning contractions, of course 😉).

Now I find myself asking quite often "How would Maira write this?" 😂